connect/runtime/src/main/java/org/apache/kafka/connect/util/ConnectUtils.java

```diff
@@ -209,4 +209,13 @@ public static String clientIdBase(WorkerConfig config) {
         }
         return result + "-";
     }
+
+    /**
+     * Get the class name for an object in a null-safe manner.
+     * @param o the object whose class name is to be returned
+     * @return "null" if the object is null; or else the object's class name
+     */
+    public static String className(Object o) {
```

Comment:
Just a general remark, feel free to put non-Connect-specific stuff like this in the general-purpose [Utils class|https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/utils/Utils.java] instead of `ConnectUtils`. Don't have to do that for this PR but it's nice to keep in mind (especially since there's some great stuff in `Utils` that's easy to miss if you aren't in the neighborhood often).